acceptance/tests/options/config_file/ttls_cached_custom_and_core_fact.rb

```diff
@@ -0,0 +1,126 @@
+test_name 'missing facts should not invalidate cache' do
```

Comment:
The test does not cover all the cases from FACT-2857 description.
The test should
- check we can cache a fact
- check that when we query the second time for the same fact we get the fact value from the cache
- request to_hash
- check that the cache is no longer changed, no matter what fact we ask for